io/io/inc/TFileCacheRead.h

```diff
@@ -76,6 +77,8 @@ class TFileCacheRead : public TObject {
    TFileCacheRead(const TFileCacheRead &);            //cannot be copied
    TFileCacheRead& operator=(const TFileCacheRead &);
 
+   std::unique_ptr<TFileBufferRead> fBufferFile;
```

Comment:
Is it a fBufferFile or fFileBuffer or is there a more descriptive word?   Can you also add a descriptive comment?